File: app/src/main/java/com/vsoft/goodmankotlin/model/DieIdResponse.java
package com.vsoft.goodmankotlin.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class DieIdResponse {

    @SerializedName("die_id")
    @Expose
    private String dieId;
    @SerializedName("part_id")
    @Expose
    private List<String> partId = null;

    public String getDieId() {
        return dieId;
    }

    public void setDieId(String dieId) {
        this.dieId = dieId;
    }

    public List<String> getPartId() {
        return partId;
    }

    public void setPartId(List<String> partId) {
        this.partId = partId;
    }

}

File: app/src/main/java/com/vsoft/goodmankotlin/model/DieIdResponse.kt
package com.vsoft.goodmankotlin.model

import com.google.gson.annotations.SerializedName

class DieIdResponse (
    @SerializedName("_id"     ) var Id     : Id?          = Id(),
    @SerializedName("die_id"  ) var dieId  : String?      = null,
    @SerializedName("part_id" ) var partId : List<String> = arrayListOf()
)